jvm-packages/docker/Dockerfile-gpu-ubuntu14

```diff
@@ -4,24 +4,37 @@ MAINTAINER h2oai "h2o.ai"
 ARG JENKINS_UID='2117'
 ARG JENKINS_GID='2117'
 ARG H2O_BRANCH='h2o3'
+ARG PYTHON_VERSIONS='2.7,3.5,3.6'
 
+# Install required packages
 RUN \
     apt-get update && \
     apt-get install -y software-properties-common && \
     add-apt-repository -y ppa:webupd8team/java && \
+    add-apt-repository -y ppa:deadsnakes && \
     apt-get update && \
     echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections && \
-    apt-get install -y zip oracle-java8-installer wget git unzip python
+    DEBIAN_FRONENT=noninteractive apt-get install -y zip oracle-java8-installer wget git unzip python3 python3-dev \
```

Comment:
There seems to be a typo here. Should it really be fronent?

Reply:
Yes, typo. Thanks!